src/validations/super-admins.js

```diff
@@ -0,0 +1,41 @@
+const Joi = require('joi');
+
+const emailRegex = /^[^\s@]+@[^\s@]+.[^\s@]+$/;
+const passRegex = /^(?=.*\d)(?=.*[A-Z]).{8,}$/;
+
+const validateCreation = (req, res, next) => {
```

Comment:
The name of this const could be validateSuperCreation for consistency with the other resources of the app